internal/exec/util/file.go

```diff
@@ -112,6 +114,33 @@ func RenderFile(l *log.Logger, c *resource.HttpClient, f types.File) *File {
 		return nil
 	}
 
+	if f.User.Name != "" {
+		u, err := user.Lookup(f.User.Name)
```

Comment:
I was meaning the one already in ignition. Seems reasonable to export it.

https://github.com/coreos/ignition/blob/b311078c6c4ddea72f54c894813f91cc8b10c3da/internal/exec/util/user_lookup.go#L27-L28